eng/Versions.props

```diff
@@ -130,14 +132,19 @@
     <MicrosoftExtensionsHttpPollyLTSVersion>8.0.13</MicrosoftExtensionsHttpPollyLTSVersion>
     <MicrosoftExtensionsObjectPoolLTSVersion>8.0.13</MicrosoftExtensionsObjectPoolLTSVersion>
     <!-- 3rd party dependencies -->
+    <AspireVersion>9.1.0</AspireVersion>
     <AzureAIProjectsVersion>1.0.0-beta.3</AzureAIProjectsVersion>
     <AzureAIOpenAIVersion>2.2.0-beta.1</AzureAIOpenAIVersion>
     <AzureIdentityVersion>1.13.2</AzureIdentityVersion>
     <AzureSearchDocumentsVersion>11.6.0</AzureSearchDocumentsVersion>
+    <CommunityToolkitAspireHostingOllamaVersion>9.2.2-beta.215</CommunityToolkitAspireHostingOllamaVersion>
+    <CommunityToolkitAspireOllamaSharpVersion>9.2.2-beta.215</CommunityToolkitAspireOllamaSharpVersion>
     <MicrosoftSemanticKernelConnectorsAzureAISearchVersion>1.41.0-preview</MicrosoftSemanticKernelConnectorsAzureAISearchVersion>
+    <MicrosoftSemanticKernelConnectorsQdrantVersion>1.41.0-preview</MicrosoftSemanticKernelConnectorsQdrantVersion>
     <MicrosoftSemanticKernelCoreVersion>1.41.0</MicrosoftSemanticKernelCoreVersion>
     <OllamaSharpVersion>5.0.7</OllamaSharpVersion>
     <OpenAIVersion>2.2.0-beta.1</OpenAIVersion>
+    <OpenTelemetryVersion>1.9.0</OpenTelemetryVersion>
```

Comment:
Please note this will result in a conflict after https://github.com/dotnet/extensions/pull/6108